### DISCUSSION TITLE
Filepond translation

### DISCUSSION THREAD

**Discussion by aliloubm** [filamentphp/filament#893] [Help]
Hello, 

How to add translation option for filepond ? 

filepond documentation 
https://github.com/pqina/filepond

```
import pt_BR from 'filepond/locale/pt-br.js';

FilePond.setOptions(pt_BR)
```

**Comment by underdpt** [comment]
I'm also dealing with this, but `FilePond` is not defined. Any help?

**Reply by danharrin** [reply]
This should be fixed in the latest release without any extra work (just set your app's locale), is it working?

**Reply by underdpt** [reply]
Hello,

I'm on latest version (v2.9.19) and tried with `'locale' => 'es',` on `config/app.php` without success. Also tried with `es_ES` and `es-es` (the spanish translation file on filepond is `es-es.js`

I've taken a look at the latest commits and saw nothing about this recently.

**Reply by danharrin** [reply]
Whoops, I thought I read DatePicker not FilePond 🤦 We don't have translation support for that yet
